### DISCUSSION TITLE
Styling the record clippings.

### DISCUSSION THREAD

**Discussion by DragosJipa** [berviantoleo/react-multi-crop#2419] [General]
Styling the record clippings doesn't work. I can't modify anything although I should be able.
The only thing that works is setting the cornerColor, cropBackgroundColor, etc... on  <ReactMultiCrop> which display as deprecated attributes. But those are the only ones that work. The problem is that I need to style 2 clippings differently...

**Comment by berviantoleo** [comment]
Currently, it doesn't support styling those clippings differently. I'll need to figure out an effective way to style those clips.

**Reply by DragosJipa** [reply]
Is it something that can be expected in the near future or is it out of scope for now?

Thanks for the reply btw

**Reply by berviantoleo** [reply]
Yes, it would be a great feature. However, I don't have enough time to maintain the new feature.

**Reply by berviantoleo** [reply]
I made initial changes to handle this case.

https://github.com/berviantoleo/react-multi-crop/pull/2504

I believe the current solution/workaround is not the best. I'll need to handle some cases, like the new clips (if needed).
